pkgs/top-level/aliases.nix

```diff
@@ -1743,6 +1743,8 @@ mapAliases ({
   syncthing-cli = syncthing; # Added 2021-04-06
   synology-drive = throw "synology-drive has been superseded by synology-drive-client"; # Added 2021-11-26
   system_config_printer = throw "'system_config_printer' has been renamed to/replaced by 'system-config-printer'"; # Converted to throw 2022-02-22
+  systemdStage1 = systemd;
+  systemdStage1Network = systemd;
```

Comment:
I'd just drop theses aliases entirely, these were only supposed to be used in as a systemd stage1, and you already refactored all call sites (and this is what's currently conflicting the PR)

Reply:
Fair enough.